moonbeam-types-bundle/index.ts

```diff
@@ -561,6 +561,10 @@ export const moonbeamDefinitions = {
           account: "AccountId",
           deposit: "Balance",
         },
+        ParachainBondConfig: {
+          account: "AccountId",
+          percent: "Percent",
```

Comment:
Is "Percent" a valid type? If it isn't, will the CI tell us if this type is not being used in a TS test?

Reply:
It seems it does, you can try on polkadotjs directly also